online.hatsune_miku.tachidesk-vaadinui.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "online.hatsune_miku.tachidesk-vaadinui",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "vaaui.sh",
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/app/bin/",
+    "--share=network",
+    "--filesystem=~/.TachideskVaadinUI/",
+    "--socket=x11",
+    "--share=ipc"
+  ],
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk17/install.sh"
+      ]
+    },
+    {
+      "name": "Tachidesk-VaadinUI",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 tachidesk-vaadinui.sh /app/bin/vaaui.sh",
+        "mv Tachidesk-VaadinUI.jar /app/bin/Tachidesk-VaadinUI.jar",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.desktop /app/share/applications/online.hatsune_miku.tachidesk-vaadinui.desktop",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.png /app/share/icons/hicolor/128x128/apps/online.hatsune_miku.tachidesk-vaadinui.png",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.appdata.xml /app/share/metainfo/online.hatsune_miku.tachidesk-vaadinui.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://github.com/Suwayomi/Tachidesk-VaadinUI/releases/download/v1.0.3/Tachidesk-VaadinUI-1.0.3.jar",
+          "dest-filename": "Tachidesk-VaadinUI.jar",
+          "sha256": "f4ed7c0537be341f22443c49a0dd0c3afad001f1eb8adb08538963875e1b323d"
+        },
+        {
+          "type": "file",
+          "path": "tachidesk-vaadinui.sh"
+        },
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/Suwayomi/Tachidesk-VaadinUI/f08d40d1dfa82e8aaea3d8f933b1547caf44fb4a/linux/flatpak/online.hatsune_miku.tachidesk-vaadinui.desktop",
+          "dest-filename": "online.hatsune_miku.tachidesk-vaadinui.desktop",
```

Comment:
```suggestion
```

Remove it from the appdata and icon too. Here dest-filename is exactly the basename of the URL.